pkgs/os-specific/linux/lvm2/common.nix

```diff
@@ -52,7 +52,7 @@ stdenv.mkDerivation rec {
     "--with-default-run-dir=/run/lvm"
     "--with-systemdsystemunitdir=${placeholder "out"}/lib/systemd/system"
     "--with-systemd-run=/run/current-system/systemd/bin/systemd-run"
-  ] ++ lib.optionals (!enableCmdlib) [
+  ] ++ lib.optionals (!enableCmdlib && !onlyLib) [
     "--bindir=${placeholder "bin"}/bin"
     "--sbindir=${placeholder "bin"}/bin"
     "--libdir=${placeholder "lib"}/lib"
```

Comment:
Is the `--libdir` flag not required?

Reply:
I don't believe so, those outputs are disabled if `onlyLib` is `true`:
https://github.com/NixOS/nixpkgs/blob/e61a2da3885cf5846476b643152cedc86516d802/pkgs/os-specific/linux/lvm2/common.nix#L137-L140

so having a placeholder doesn't make sense. It's been a while since I looked at this so I'm not 100% sure.